services/terrain/src/terrain/core.clj

```diff
@@ -59,6 +60,12 @@
         (tc/with-logging-context {:user-info (cheshire/encode user-info)}
                                  (handler request))))))
 
+(defn- wrap-tree-urls-config
+  [handler]
+  (fn [request]
+    (with-tree-urls-base (config/tree-urls-base)
+      (handler request))))
```

Comment:
Ah, that's a good idea; that'd make it so I don't need to load the core namespace or `with-tree-urls-base` into the core namespaces of the services, which was one of the things that made me mildly uncomfortable about this setup.